### PR TITLE
add RADAR section

### LINKED ISSUE #17: write RADAR section

### PR DESCRIPTION
**Pull request by LeviUniAcc**
the citation does not work, can maybe someone show me how it goes :) i tryed to do it like this https://www.overleaf.com/learn/latex/Bibliography_management_with_bibtex but it does not work.
@RaphaW @EhrenbruderRoschtbraten

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -73,6 +73,9 @@ \subsubsection{Text detector models}
 
 The \textbf{RoBERTa Base OpenAI Detector} is the output detector model of GPT-2. It was obtained by fine-tuning a RoBERTa base model with the outputs of the GPT-2 model containing 1,5 Billion parameters. The RoBERTa Base OpenAI Detector can be used to classify texts, on whether they are likely to be created by the GPT-2 model. It was published by OpenAI, when the weights of the GPT-2 model 1,5 Billion parameter version was released.
 
+The \textbf{RADAR-Vicuna-7B} text detector is an adversarial-learning trained model introduced by TrustSafeAI. In the training stage the detector tryed to differentiate between human and ai text while a paraphraser trained to make the text more humanly.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]: I would write AI always in capital letters as it is a name
```suggestion
The \textbf{RADAR-Vicuna-7B} text detector is an adversarial-learning trained model introduced by TrustSafeAI. In the training stage the detector tryed to differentiate between human and AI text while a paraphraser trained to make the text more human.
```

File: paper.tex [hunk 2]
```diff
@@ -73,6 +73,9 @@ \subsubsection{Text detector models}
...
+The Ai-generated text for learning is extracted by the LLM Vicuna-7B-v1.1 written by LMSYS Org \cite{hu2023radar}.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
The AI-generated text for learning is extracted by the LLM Vicuna-7B-v1.1 written by LMSYS Org \cite{hu2023radar}.
```

**Review by EhrenbruderRoschtbraten — CHANGES_REQUESTED** [review]
Some small changes

File: paper.tex [hunk 3]
```diff
@@ -73,6 +73,9 @@ \subsubsection{Text detector models}
 
 The \textbf{RoBERTa Base OpenAI Detector} is the output detector model of GPT-2. It was obtained by fine-tuning a RoBERTa base model with the outputs of the GPT-2 model containing 1,5 Billion parameters. The RoBERTa Base OpenAI Detector can be used to classify texts, on whether they are likely to be created by the GPT-2 model. It was published by OpenAI, when the weights of the GPT-2 model 1,5 Billion parameter version was released.
 
+The \textbf{RADAR-Vicuna-7B} text detector is an adversarial-learning trained model introduced by TrustSafeAI. In the training stage the detector tryed to differentiate between human and AI text while a paraphraser trained to make the text more human.
```
Comment (1Maxnet1) [inline paper.tex]: I would propose to fix the typo and replace the the with an a to reduce the amounts of "the" in the sentence.
```suggestion
The \textbf{RADAR-Vicuna-7B} text detector is an adversarial-learning trained model introduced by TrustSafeAI. In the training stage a detector tried to differentiate between human and AI text while a paraphraser trained to make the text more human.
```

**Review by 1Maxnet1 — APPROVED** [review]
LGTM, two remaining issues.

**Review by RaphaW — APPROVED** [review]
LGTM, cannot see the citation rn, but we can fix that later

**Pull request merged**